src/store/modules/user/index.js

```diff
@@ -197,9 +197,20 @@ const actions = {
    *
```

Comment:
I've added support of change-password errors to be passed to user from the backend. For example, `Worng old password`